src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.wallet.events
+  (:require [taoensso.timbre :as log]
+            [re-frame.core :as re-frame]))
+
+(re-frame/reg-event-fx :wallet-2/get-wallet-token
```

Comment:
rf/defn

Reply:
wait, I thought we said we would try out not using rf/defn for a while? or is there some scope on where we trial using that for the moment? 🤔 
cc @ilmotta

Reply:
i feel like it will be only confusing if we'll try to mix

Reply:
mix what exactly? the reg-event-fx and rpc calls?

Reply:
mix re-frame/reg-event-fx and rf/defn

Reply:
I did it as part of https://github.com/status-im/status-mobile/pull/17451

Reply:
ah okay, I think we should just add `re-frame/reg-event-fx` to utils.re-frame namespace then?
so we can call `rf/reg-event-fx` 🤔

Reply:
lets wait Icaro :)

Reply:
@ilmotta Please check here

Reply:
Hey guys. I agree mixing APIs can be confusing, but this is going to be temporary. I'm going to start refactoring events in `status_im2/` with the help of others (if possible) next week. It's not a huge amount of work, but has to be done carefully.

I think using `rf/defn` to avoid mixing APIs is good too, let's just be aware that this will increase the work to refactor soon. Maybe the ideal situation is to refactor entire namespaces to make sure at least on the same file we're consistent, i.e. every `events.cljs` file should either use `rf/defn` or the `reg-event-fx` with `:fx`, but not both. Sometimes this is impossible if the `events` file is too big.